vast.yaml.example

```diff
@@ -62,8 +62,10 @@ vast:
   # Additional directories to load plugins specified using `vast.plugins` from.
   plugin-dirs: []
   # The plugins to load at startup. For relative paths, VAST tries to find the
-  # files in the specified `vast.plugin-dirs`.
-  # Note: Add `example` or `/path/to/libexample.so` to load the example plugin.
+  # files in the specified `vast.plugin-dirs`. The special values 'bundled' and
+  # 'all' enable autoloading of bundled and all plugins respectively.
```

Comment:
This should probably clarify what "all" plugins means, ie. that it will attempt to load all shared libraries matching `libvast-plugin-*` from the configured `plugin-dirs`.